pmacApp/src/pmacController.cpp

```diff
@@ -4652,7 +4696,7 @@ asynStatus pmacController::tScanCalculateVelocityArray(double *positions, double
   int prevBuffLastVelMode = -1;
   double prevBuffPosition = 0;
   double prevBuffVelocity = 0;
-  int prevBuffTime = 0;
+  double prevBuffTime = 0;
```

Comment:
Note for myself: not related with this PR, and can't remember why I have changed. Possibly will revert this.